returnn/search.py

```diff
@@ -15,7 +15,7 @@
 import os
 import shutil
 import subprocess as sp
-from typing import Any, Union, Set, Dict
+from typing import Optional, Any, Union, Set, Dict
```

Comment:
```suggestion
from typing import Any, Union, Set, Dict
```
import not needed any more